configuration/company/res.company.info.csv

```diff
@@ -0,0 +1,2 @@
q5w0bIyXYLkVb5wLkNyfSP5fQkkmzeS35dQT15Jp1scaL4ISf0q3jhw4g+FuFDL/CrGeKFmcgeV8BnqFLbMUUmqmt25VA4roWWOSRJmrWGsFmYXoJgkbq4/gSRa6mhqaWUl6619VEMXIikk2MRcucNbwv3JLpTJc8yTxBye5Hd445J8v0x22b+UJSIeqyL5XMkniZvxWPsvn54CNSnqKLWEulMe3opIOphWDwJJzH0J/CSm/YJnCKqF+czTExgRzP7kSdaHiCT5SUwbsWUHp8Ic+8ZPRLOPPEkdMReAsNaj/oToUq04xz7iKwcqv4RMH3gtZpjwwf8Ly0z2LBLhfBy0gQU4GQJJxIzm4gkaBlpFmTkfh28T4fW8SYPBERazoI1QSoc24TVI4FE4s+lJOkcMyyqtnHkH8ZYOCCGpuxXzhGMINgI4p6ad+As4R9tp4SioKxQ7pKAUj2xi56kiGRIPmDZO0nmL5h2CXQB2dKV4cCVOTQkzX7Dym0THK4qhR/HoStPT4Flga5bfEgLtlVkVz1tEKd/y7orw5Mqn0kcpAXbSSd3PoMGdBHdX7oYWgE1UDhwdG6W8bJhUk8zw4lAqS4Dqa865tJonX6zceWWNNZP7HB6TX08+UN56KNlBOV1QeAQoPY18Vy3RpWotcyK6QduPsDPky5XHlzKOugbvIJzf/SuThaa1L9HBVSDvQEtPnXaM17tDGftVtOrP9+z53QYbkJQ64qFh06DHr9/ffVv397vsmfKuLrtqrem2rvPl7QdBNrEGfxK645iVxfvu/hyfau/Pb2bemfJGOJsiDqJzpo2z5GSKQOYAbz3catgajB/O0HI7X7tWToKSNCQTl872KY9UOtsgNY0QlQlK5nD+xsU/UEF+nB+fP1A5ekvqmI35+adLUaamRJUip/hzKuFOFcqc8o2lTNnGktCbgZoilIuWKpT2SI1HpAEutD4ybXYcGScI1KC0W9KQNpZMeUZWRVQ6lRNpr7gqUBKQpPqv4L9JzewiATgVvOJLk7GgrS7w1vgZrKxR2FJ+Rg4Px8kbSIbqk4nI0iGNrrD5ChPkBC11awtRxyfbrytT5lP++iaA0zMWS8N8TycuoYOaWM0O0kluqD6auf2O8+WYi1NhV5ewfDdKh0n7aTthNAa8QRcpaWLd1peCKTjU5E5rGS4Zxnsxx9Cw97J7rpglqvJU24BsFLxxUH4Gyrrtrx/kQUFie1N7Ktel593LF/JOsBGBamkKR1DyzDTGo7r/UTKbP5RfmOVGkUvFOEvRJzamf5zNpjDLzVfMIbt0SZXn3/+pgzfddNNNN91000033XTTTTfddNNNfyuxYRZxqIX/X3YwpmB8Juc+0VUx6XP0Me5uOZaLrsqXQBw/T/+UGUZK/yg+ffYBvymA2cTtdvXhKr0XLNPzAOAYn/7VGoenVgB3Hhi3FQymc1fX3Ym4JROmg2bd1pvbeGoJvGVL79aDefQ5OLrc1h3Yu2xRNcyFq313ya4GySY+coneAKbuJH3sEUuuAx3yXdh36fMFTfaY5DqMWQ4j2AfJ0+tLdodjoHd1d7g6F2R17W8bmjZddrlnt/RkvT1M6h7TfrX2GV/wj+DFSm80a0+m5D7HHr+NFo7t0czTvE6rvQx/ho4VrKqatp/3wuyzMLG5tf3ghjseYWSv+avhZCJYSeI7mPXotBg0fD/JYgstl33qoUchs6fbtuPHhIHP/thvrzTu4WHVG+01T5s+BN3zNisBPbGHFQataXux+91pO+YBE92LS8GInUwMVt72TuiOwvvMfe7ioD2LitaYEYZ+bx4VvobPkn2S+/vbtADYMUndT1b8w9y2Q5I2R7IdkRwkKGOSq3ZcLmDcjmd9UpL82elQ7Q2iB4TzzomBTjsux1Bth4TI3Zcwr8efmCQ50sjT0w8XkJyHq8rJh9LyDXLAkB8jubBt00IAE3fbi0oDNHorPyIZDNrx1T25WkiyPWtE0V1AMv5CPsl+u6QN4rD1k7nxpFof1ckPFAcKjUnJ91ySSY0+sVKQ1ahMJ38HN8k6S5zoM5JdUvFG3w3W0I5jEoKo1NPvtD8gSUiHl79DstEblzOtQe5flS81aDmEQZRehZCMXnTeuH0Ma6sPNm/DoJemwIBpW9oxD/okQ7JOPhbio2HYS0hSJ+k/j773MUnYt9n1b5Ckz0sr8vPf3TQm5UstFiFe9NQw77aYmuRVEV7p6t1QvrxNBGDS5nKJ5JB0c0hSh4nvhMoSuJhkYGnZfoCPSdL/0D98jaQVP73NPIfGMbBttzVanbe5htxfemOxypC2+MY+q6dgrjemRBN/Fpdfa+5FE8Pyd7WWze9G/mJMarRhRWw2aWMlJamx8g0BXZX2MckI5RdjchU/ffQHut9h2l8G3bMSQQilO6zJgnVkSR9F79awNRu4XkxykD9tCl4QCCuSCMlMPTnMJTklFceW/nQCyQb5z4BW5J+RDFF+uXRnn57+v9U9i+SqnfwSMCIBBA++G0W/7rNnjd44ULVaKcnce031llRCoJ26Pp6wAlhPLjwwhxlJ0hdoP7BmD0+SlosHFtUxye5I/nZEMkT5jXoyfWDua9+S1x7GnaY5a4zoabclWIeehmWYtIWP4ScL3t19fyJfJ42e6L1Fqjv6bdJqjvpSoLlhp4ORJIh7YQeFJ6nBuhc2OiOSA+Ks+O2YJHu84TdINpJe5ESu6P5MxP9mn8Z2o9UjJkhDJ/lhoMOuJK0g4kcnfndn+93QITGVKTfe2lp67L3fCzHDsTeny1K0ju5WOZKaN6vmkKzGV8M2GQzbY9ZHTr7dSNvDMz9MavsZyRn39DD+x2jP/jmd9ETEf47S2wZ2ENT1I62PphYXV7B2QWhPDtn7BNLUKFwKFmi7zYBJ6DFDp2W7gWsNE4cbk64eBPY62tMYk4yKmUgyvRq3bmdDcrvAbkYr/ziSBJL9FZLpKll7CWy0gLhP/J9/lrnjfwYbdwI9cHsgAAAAAElFTkSuQmCC"
```

Comment:
Let's take the public contact email address of C2C

Reply:
Should we use the transparent background logo?

Reply:
I can't seem to find the C2C logo with a transparent bg